plonk-core/src/proof_system/widget/arithmetic.rs

```diff
@@ -43,23 +45,20 @@ where
 
 impl<F> ProverKey<F>
 where
-    F: FftField,
```

Comment:
`PrimeField` implements `FftField` which implements `Field`. The most general API is `Field`, `FftField`, or `PrimeField` in increasing specificity. I recommend `Field` since it is the most general, and we only need `FftField` and `PrimeField` in some places.